examples/visualbasic/OpenGL.vb

```diff
@@ -7,37 +6,38 @@ Imports OpenTK
 Imports OpenTK.Graphics
 Imports OpenTk.Graphics.OpenGL
 
-Module OpenGL
-
-    Dim WithEvents window As RenderWindow
+Friend Module OpenGL
+    Private WithEvents window As RenderWindow
 
     ''' <summary>
     ''' Entry point of application
     ''' </summary>
-    Sub Main()
+    Public Sub Main()
 
         ' Request a 24-bits depth bufer when creating the window
-        Dim contextSettings As New ContextSettings()
-        contextSettings.DepthBits = 24
+        Dim contextSettings As New ContextSettings With {
+            .DepthBits = 24
+        }
 
         ' Create main window
         window = New RenderWindow(New VideoMode(800, 600), "SFML graphics with OpenGL (Visual Basic)", Styles.Default, contextSettings)
         window.SetVerticalSyncEnabled(True)
 
         ' Initialize OpenTK
-        Toolkit.Init()
-        Dim context As GraphicsContext = New GraphicsContext(New ContextHandle(IntPtr.Zero), Nothing)
+        Dim unused = Toolkit.Init()
```

Comment:
maybe it can be `unused1` and alter `unused2` instead of `unused` and `unused1`

Reply:
Changed them to 1 & 2.